lib/active_merchant/billing/gateways/merchant_warrior.rb

```diff
@@ -60,9 +60,12 @@ def refund(money, identification, options = {})
         commit('refundCard', post)
       end
 
-      def void(money, identification, options = {})
+      def void(identification, options = {})
         post = {}
-        add_amount(post, money, options)
+        # The amount parameter is required for void transactions
+        # on the Merchant Warrior gateway.
+        post['transactionAmount'] = options[:amount]
```

Comment:
Is it okay that we're not setting `transactionCurrency` anymore in here?

Reply:
I believe so. When I initially added the void transaction to Merchant Warrior, they sent us this information for the fields that are needed for void:
![image](https://user-images.githubusercontent.com/42787645/74677849-6c70fb80-5187-11ea-9db5-893fefcef459.png)

Reply:
Interesting, not a `currency` in sight.